public/electron-starter.js

```diff
@@ -34,7 +34,7 @@ function createWindow() {
   }));
 
   // Open the DevTools.
-  mainWindow.webContents.openDevTools({ mode: 'detach' });
+  mainWindow.webContents.openDevTools(); //{ mode: 'detach' });
```

Comment:
What's the thinking with this? I think ultimately we want to have a menu button or a key combination (or both) during alpha that opens the dev tools. Perhaps get rid of this, and I'll put up an issue for that?